.gitignore

```diff
@@ -28,3 +28,6 @@
 node_modules
 /web/themes/custom/contribtracker/dist/
 .DS_Store
+
+# Ignore drupal-quality-checker generated config
+/*.dist
```

Comment:
This is already in the file now at the top.

Reply:
Ahh missed it. Removed it